tools/downloader/topologies.py

```diff
@@ -0,0 +1,75 @@
+import os
+import argparse
+import sys
+
+from pathlib import Path
+from argparse import Namespace
+
+import common
+import downloader
+import converter
+import info_dumper
+
+_cache = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', Path('.').resolve() / 'models'))
+
+def __load(model_name):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--name', metavar='PAT[,PAT...]',
+        help='only dump info for models whose names match at least one of the specified patterns')
+    parser.add_argument('--list', type=Path, metavar='FILE.LST',
+        help='only dump info for models whose names match at least one of the patterns in the specified file')
+    parser.add_argument('--all', action='store_true', help='dump info for all available models')
+    parser.add_argument('--print_all', action='store_true', help='print all available models')
+    args = Namespace(all=False, list=None, name='mobilenet-ssd', print_all=False)   
+            
+    return common.load_models_from_args(parser, args)[0]
+
+def load_model(model_name, precision='FP32'):
+    download_dir = _cache / __load(model_name).subdirectory
+
+    downloader.main(['--name=' + model_name, '--precisions=' + precision,
+                '--output_dir=' + str(_cache).replace('\\', '\\\\')])
+        
+    return Topology(model_name, download_dir, precision)
+
+def help(model_name):
+    _topology = __load(model_name)
+    description_res = 'Model description for ' + model_name +' \n  ' + \
+                _topology.description + '\n     License: ' +  _topology.license_url
+
+    print(description_res)
```

Comment:
Please show example of `help(ssd)`

Reply:
![image](https://user-images.githubusercontent.com/32199725/88289230-98c41180-ccfd-11ea-8f61-6695026c94a3.png)

Reply:
We  should use it as ```omz.help('model-name')```

Reply:
But what about help for `class Topology`?

Reply:
![image](https://user-images.githubusercontent.com/32199725/88399484-9fbb5480-cdcf-11ea-8878-5b3b4d23220f.png)